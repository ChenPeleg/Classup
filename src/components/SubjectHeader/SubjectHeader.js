import React from "react";
import "./SubjectHeader.scss";
const subjectHeader = (props) => <p className="subjectHeader">{props.text}</p>;

export default subjectHeader;
