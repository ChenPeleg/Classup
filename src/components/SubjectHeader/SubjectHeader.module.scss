@import "../../assets/styles/base";
@font-face {
  font-family: "LuckiestGuy";
  font-weight: 400;
  font-style: normal;
  src: url("../../assets/fonts/Luckiest_Guy/LuckiestGuy-Regular.ttf")
    format("truetype");
}
.subjectHeader {
  margin: 0px;
  display: block;
  align-self: center;
  font-family: LuckiestGuy;
  color: yellow;
  display: inline-block;

  font-size: $base_Header_Size;
  font-weight: bold;
  letter-spacing: 2px;
  background: linear-gradient(
    90deg,
    rgba(2, 0, 36, 1) 0%,
    rgba(126, 250, 181, 1) 0%,
    rgba(249, 165, 236, 1) 21%,
    rgba(249, 249, 159, 1) 44%,
    rgba(190, 221, 255, 1) 73%,
    rgba(255, 255, 146, 1) 91%,
    rgba(232, 225, 84, 1) 100%
  );
  -webkit-text-stroke-width: 1px;
  -webkit-text-stroke-color: black;
  -webkit-background-clip: text;
  -webkit-text-fill-color: transparent;
  @include mobile {
    font-size: $Mobile_fontSize;
  }
}
