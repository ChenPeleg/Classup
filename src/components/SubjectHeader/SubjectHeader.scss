@font-face {
  font-family: "LuckiestGuy";
  font-weight: 400;
  font-style: normal;
  src: url("../../components/styles/Luckiest_Guy/LuckiestGuy-Regular.ttf")
    format("truetype");
}
.subjectHeader {
  font-family: LuckiestGuy;
  color: yellow;
  display: inline-block;
  margin: 0px;
  font-size: 26px;
  font-weight: bold;
  letter-spacing: 2px;
  color: black;
  -webkit-text-fill-color: white; /* Will override color (regardless of order) */
  -webkit-text-stroke-width: 1px;
  -webkit-text-stroke-color: black;
}
