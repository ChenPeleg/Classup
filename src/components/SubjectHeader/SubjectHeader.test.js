import React from "react";
import { shallow } from "enzyme";
import SubjectHeader from "./SubjectHeader";
const wrap = shallow(<SubjectHeader />)

describe("SubjectHeader", () => {
    it("renders correctly", () => { expect(wrap).toMatchSnapshot() })
})