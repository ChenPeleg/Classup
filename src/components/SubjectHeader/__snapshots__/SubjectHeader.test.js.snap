// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubjectHeader renders correctly 1`] = `ShallowWrapper {}`;
